java/test/src/main/java/io/ray/test/GetTimeoutTest.java

```diff
@@ -0,0 +1,87 @@
+package io.ray.test;
+
+import io.ray.api.ActorHandle;
+import io.ray.api.ObjectRef;
+import io.ray.api.Ray;
+import io.ray.runtime.exception.RayTimeoutException;
+import java.util.ArrayList;
+import java.util.List;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+@Test
+public class GetTimeoutTest extends BaseTest {
+
+  public static int squareDelay(int x) {
+    try {
+      Thread.sleep(5 * 1000);
+    } catch (Exception e) {
+      e.printStackTrace();
+    }
+    return x * x;
+  }
+
+  public static class Counter {
+
+    private int value;
+
+    public Counter(int initValue) {
+      this.value = initValue;
+    }
+
+    public int getValueDelay() {
+      try {
+        Thread.sleep(5 * 1000);
+      } catch (Exception e) {
+        e.printStackTrace();
+      }
+      return value;
+    }
+
+    public int getValue() {
+      return value;
+    }
+  }
+
+  private void checkTimeout(long startTime, long timeout) {
+    long waitTime = System.currentTimeMillis() - startTime;
+    System.out.printf("Except wait %d ms, actual spend %d ms.%n", timeout, waitTime);
+    Assert.assertTrue(Math.abs(waitTime - timeout) < 1500);
+  }
+
+  @Test
+  public void testActorGetTimeout() {
+    ActorHandle<Counter> actor = Ray.actor(Counter::new, 1).remote();
+    Assert.assertEquals(Integer.valueOf(1), actor.task(Counter::getValueDelay).remote().get());
+    Assert.assertEquals(Integer.valueOf(1), actor.task(Counter::getValueDelay).remote().get(10000));
+
+    long startTime = System.currentTimeMillis();
+    long timeout = 2000;
+    Assert.assertThrows(
+        RayTimeoutException.class,
+        () -> Ray.get(actor.task(Counter::getValueDelay).remote(), timeout));
+    checkTimeout(startTime, timeout);
+  }
+
+  @Test
```

Comment:
```suggestion

```
Since we added the `@Test` annotation onto the class, there is no need to add it to every method.